py/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.8.24
+
+- Add optional event to respect signature of the `get_sampling_result` method. ([#2026](https://github.com/getsentry/relay/pull/2026))
```

Comment:
I would remove this changelog entry, because the observable behavior of the exposed python function does not change (or does it?). If we keep it, please move it to `## Unreleased`.